index.js

```diff
@@ -66,7 +67,7 @@ function fastErrorHTTP(code, status) {
  */
 
 // NOTE: next is needed, even if not used, per https://expressjs.com/en/guide/using-middleware.html
-function showError(err, req, res, next) { // eslint-disable-line no-unused-vars
+function showError(err, req, res, next, ttl) { // eslint-disable-line no-unused-vars
```

Comment:
next is not used?

Reply:
Yep. See the comment on line 69.

Reply:
Yeah, I was just wondering why we keep it if it's not used.